public/unity/webgl.json

```diff
@@ -0,0 +1,13 @@
+{
+"companyName": "DefaultCompany",
+"productName": "falling squares",
+"dataUrl": "webgl.data.unityweb",
+"wasmCodeUrl": "webgl.wasm.code.unityweb",
```

Comment:
Is there source code for this?

Reply:
there is. Not sure where it should live; I'm thinking a separate repo